pkg/virt-api/rest/subresource.go

```diff
@@ -469,7 +469,26 @@ func (app *SubresourceAPIApp) StartVMRequestHandler(request *restful.Request, re
 		return
 	}
 
-	patchType := types.MergePatchType
+	startPaused := false
+	startChangeRequestData := make(map[string]string)
+	if vm.Spec.Template != nil && vm.Spec.Template.Spec.StartStrategy == nil {
```

Comment:
My approach was to honor what is explicitly set in VM/VMI definition and not to override. If the field and the flag are both set, it will start paused and we don't need to patch the status.
@davidvossel what do you think?

Reply:
i agree with @zcahana.  If the user explicitly requests to start the VM paused, then we should start it paused that first time. 

Subsequent restarts will then use whatever is in the vmi template's startStrategy.

Reply:
Changed